### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by jonnyhotchkiss** [Androidacy/MagiskModuleManager#93] [General]
I've spent so long reading references to magisk manager, and the core app, vs the downloads section... I wasn't sure if I'd done it wrong, got the wrong one, or what... ;til I just read your explanation that this search+download support has been discontinued. THANK YOU for clarifying, and providing this workaround! 

Am I right in thinking this is one of a kind? I installed obtainium to test that as a one-stop-shop for repo + app-source management, assume that doesn't do modules?

**Comment by androidacy-user** [comment]
We have the only app with support for our repo, which currently is one of two repos with any serious amount of modules. The alt repo is of questionable quality as they have very few restrictions but is available if you want to use it. AMM is the only app afaik that supports the original format, and we'd like to think it's the best one available :)

Tl;Dr: John Wu discontinued official support as he grew tired of dealing with it and the associated drama. AMM is a continuation of FoxMMM as Fox2Code unfortunately has poor health and could not continue maintaining.

AFAIK obtanium is app updates only.
